libraries/botbuilder-dialogs-adaptive-testing/src/actions/assertCondition.ts

```diff
@@ -20,6 +23,12 @@ export class AssertCondition<O extends object = {}> extends Dialog<O> {
      */
     public description: StringExpression;
 
+    /**
+     * Begins the dialog.
+     * @param dc The DialogContext.
+     * @param options Options.
+     * @returns A `Promise<DialogTurnResult>` object.
```

Comment:
```
* @param dc The DialogContext for the current turn of the conversation.
* @param options Additional information to pass to the prompt being started.
* @returns A Promise representing the asynchronous operation.
```

This is the comment from beginDialog in `activityPrompt.ts`, I think it sounds better